### PR TITLE
Fix truffle/graalvm terminology

### PR DESCRIPTION
**Pull request by bhoran**

### CODE REVIEW

File: docs/graalpy-embedded/reference/Interoperability.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+title: Interoperability with Other Graal Languages
 permalink: /docs/polyglot-api/
 toc: true
 ---
 
 # {{ page.title }}
 
-As well as a runtime to embed Python code in your Java application, GraalPy is interoperable with other languages supported by GraalVM (languages implemented with the [Truffle framework](https://www.graalvm.org/latest/graalvm-as-a-platform/language-implementation-framework/)).
-This means that you can use GraalPy to embed the code from those languages into your Python application.
+As well as a runtime to embed Python code in your Java application, GraalPy is interoperable with other Graal languages (languages implemented with the [Truffle framework](https://www.graalvm.org/latest/graalvm-as-a-platform/language-implementation-framework/)).
```
Comment (olyagpl) [inline docs/graalpy-embedded/reference/Interoperability.md:10]: Maybe we can simplify this sentence a bit, it's hard to read. For example:
Besides the well-developed interoperability with Java,  GraalPy is interoperable with other Graal languages (languages implemented with the [Truffle framework](https://www.graalvm.org/latest/graalvm-as-a-platform/language-implementation-framework/)).

Comment (olyagpl) [reply docs/graalpy-embedded/reference/Interoperability.md:10]: Another variant:

Besides being primarily recommended to use in your Java application, GraalPy can interoperate with other Graal languages (languages implemented with the [Truffle framework](https://www.graalvm.org/latest/graalvm-as-a-platform/language-implementation-framework/)).

**Review by olyagpl — APPROVED** [review]

**Comment by timfel** [comment]
I can pick this up and integrate the changes in the current main

**Comment by timfel** [comment]
@bhoran I re-applied your changes to the current main. Do you mind checking if I missed something?

**Pull request merged**